resources/js/components/modals/files/mm-modal-add.vue

```diff
@@ -33,7 +33,7 @@
             <vue-dropzone
                 ref="myVueDropzone"
                 :style="styleBtnDefault"
-                v-on:vdropzone-success="uploadSuccess($event)"
+                v-on:vdropzone-success="uploadSuccess"
```

Comment:
can the message read "saved" or just "success" instead of "uploaded" when you hit save?

Reply:
Also it feels like the modal should close after you hit save. what do you think?

Reply:
it did cross my mind but they might want to input the french values too. 

right now how it follows this flow:

1. attach image => triggers sidepanel to open
2. add english value and save 
3. toggle to french values 
4. add them and save

Reply:
makes sense!

i would just update the success message if possible then